server.js

```diff
@@ -2040,6 +2040,135 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// GitHub pullrequests integration.
+camp.route(/^\/github\/issues-pr(-raw)?\/([^\/]+)\/([^\/]+)\.(svg|png|gif|jpg|json)$/,
```

Comment:
@espadrine, I'm trying to refactor them now, but I don't think it's really a good idea, they are using the different type of apis, make them together will make the code be ugly.